src/js/components/Header/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const header =  React.createElement('div', {className: 'header'},
```

Comment:
Since Task 1 is done - we can start removing unnecessary usage of different component creation approaches.